pegasus-spark-analyser/src/main/java/com/xiaomi/infra/pegasus/spark/analyser/ColdBackupLoader.java

```diff
@@ -24,7 +24,7 @@
 
   private SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd");
 
-  public ColdBackupLoader(ColdBackupConfig config) throws FDSException {
+  public ColdBackupLoader(ColdBackupConfig config) throws PegasusException {
```

Comment:
PegasusSparkException.

The user may use pegasus-spark together with pegasus-client. We should distinguish their exceptions.